ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -292,6 +295,16 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 				on('keydown', this.handleKeyDown);
 				on('keyup', this.handleKeyUp);
 			}
+
+			if (typeof ResizeObserver === 'function') {
+				this.resizeObserver = new ResizeObserver(() => {
+					this.positionContextualPopup();
+				});
+			}
+
+			if (this.clientSiblingRef?.current) {
+				this.resizeObserver.observe(document.body);
+			}
```

Comment:
We'd better move this to the below timing since it will recalculate the position even the contexualpopup is closed.
Also, it's better to observe the real DOM that has size below "floatLayer" since we cannot gurantee that body has always its size.
We need to discuss better selectors if any.

```
getContainerNode = (node) => {
			this.containerNode = node;

			if (this.resizeObserver && typeof document !== 'undefined') {
				if (node) {
					this.resizeObserver.observe(document.querySelector(`#floatLayer > div`));
				} else {
					this.resizeObserver.disconnect();
				}
			}
		};
```

Reply:
That's a good point. This is the part I missed, and I reflected your reviews